docs/src/main/sphinx/sql/explain.rst

```diff
@@ -353,7 +353,10 @@ JSON Logical plan::
 EXPLAIN (TYPE DISTRIBUTED)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Distributed plan::
+Process the supplied query statement and create a distributed plan in text
+format. The plan takes the cluster characteristics, such a number of workers,
```

Comment:
```suggestion
format. The plan takes the cluster characteristics, such as the number of workers,
```

Reply:
`The plan takes the cluster characteristics, such as the number of workers,`

Both plans (distributed or not) take number of workers into account. It's that distributed plan shows explicitly where data is exchanged between workers (plan is split into stages).

Reply:
Updated